src/main/java/com/google/sps/servlets/UserCardsServlet.java

```diff
@@ -44,28 +45,35 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     
     // Ensures user is authenticated
     UserService userService = UserServiceFactory.getUserService();
+    List<Card> userCards = new ArrayList<>();
+
+    // Aggregate information as a json
+    Map<String, Object> jsonInfo = new HashMap<>();
+    jsonInfo.put("showCreateFormStatus", false);
+
     if (userService.isUserLoggedIn()) {
 
-        List<Card> userCards = new ArrayList<>();
+        jsonInfo.put("showCreateFormStatus", true);
 
         String folderKey = request.getParameter("folderKey");
 
-        // Query all folders identified by the userKey
         Query cardQuery = new Query("Card").setAncestor(KeyFactory.stringToKey(folderKey));
 
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         PreparedQuery results = datastore.prepare(cardQuery);
 
         if (results != null) {
           for (Entity entity: results.asIterable()) {
-            Card card = Card.EntityToCard(entity);
+            Card card = new Card(entity);
```

Comment:
If we create a card from an entity, why do we have to set the key in the next line?